_docs-sources/iac/whats-this/index.md

```diff
@@ -1,5 +1,25 @@
 # What is the Infrastructure as Code Library?
 
-Qu'est-ce que c'est?
+The Gruntwork Infrastructure as Code Library (IaC Library) is a collection of reusable code that enables you to deploy and manage infrastructure quickly and reliably. We've taken the thousands of hours we spent building infrastructure on AWS and condensed all that experience and code into pre-built packages or modules.  
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi leo urna molestie at. Volutpat blandit aliquam etiam erat. In massa tempor nec feugiat nisl pretium. Vitae auctor eu augue ut lectus arcu bibendum at. Auctor augue mauris augue neque gravida. Vulputate sapien nec sagittis aliquam malesuada bibendum arcu. Bibendum est ultricies integer quis auctor elit. Morbi tincidunt augue interdum velit euismod in pellentesque. Imperdiet proin fermentum leo vel orci porta non pulvinar neque. Felis bibendum ut tristique et egestas quis ipsum. In arcu cursus euismod quis viverra nibh cras pulvinar mattis. Massa massa ultricies mi quis hendrerit dolor. Vel pharetra vel turpis nunc eget lorem. Tortor consequat id porta nibh venenatis cras. Nunc pulvinar sapien et ligula ullamcorper. Vehicula ipsum a arcu cursus vitae congue mauris rhoncus. Sit amet porttitor eget dolor morbi non.
+The library consists of two types of code: Modules & Services
+
+## Modules
+
+Modules are reusable code to deploy and manage one piece of infrastructure and each one is a battle-tested, best-practices definition of a piece of infrastructure, such as a VPC, ECS cluster, or an Auto Scaling Group. For more information on modules check out the [Modules page](/iac/whats-this/modules/)
```

Comment:
Now is our chance to drop battle-tested and best-practices, if it no longer sparks joy :laughing: 

Seriously, I've seen it so many times across our content, I'm not sure if it's helping anyone or not. Maybe it sounds cool? What do you think?

Reply:
I agree. I pulled this phrase from existing docs and I'm not the biggest fan of it. I must've blocked it out in my brain, not realizing I used it like 4 times in these 3 files, so I'll pull that out and work on some better phrasing.